integration-cli/docker_cli_run_test.go

```diff
@@ -3174,3 +3174,14 @@ func (s *DockerSuite) TestMountIntoSys(c *check.C) {
 		c.Fatal("container should be able to mount into /sys/fs/cgroup")
 	}
 }
+
+func TestRunUnshareProc(c *check.C) {
+	testRequires(c, Apparmor)
+	defer deleteAllContainers()
+
+	name := "acidburn"
+	runCmd := exec.Command(dockerBinary, "run", "--name", name, "jess/unshare", "sh", "-c", "unshare -p -m -f -r --mount-proc=/proc mount")
```

Comment:
For some reason I've found that dropping *--mount-proc=/proc* and using *mount -t proc none /proc* works across more systems. I'm not sure if that's an issue with the versions of unshare that I've used, or the kernels.